build/mvn

```diff
@@ -163,8 +163,14 @@ export MAVEN_OPTS=${MAVEN_OPTS:-"$_COMPILE_JVM_OPTS"}
 
 echo "Using \`mvn\` from path: $MVN_BIN" 1>&2
 
-# Last, call the `mvn` command as usual
+# call the `mvn` command as usual
 "${MVN_BIN}" -DzincPort=${ZINC_PORT} "$@"
+MVN_RETCODE=$?
 
-# Try to shut down zinc explicitly
-"${ZINC_BIN}" -shutdown -port ${ZINC_PORT}
+# SPARK-25854
+# Try to shut down zinc explicitly if the server is still running.  if it's not running,
+# it's timed out and we'll still need to exit the script w/a 0 to keep the build from
+# failing.
+"${ZINC_BIN}" -shutdown -port ${ZINC_PORT} || true
```

Comment:
do we still need `|| true`? we always return `$MVN_RETCODE` now.

Reply:
we probably don't need it, but i am more than comfortable keeping it in there #justincase

Reply:
alright, i tested the script w/a `false` call in the `mvn` helper script and nothing broke.  pushed a change removing `|| true`.